### PR TITLE
Update 2022 fpl values

### PR DESCRIPTION
**Pull request by saikumar9**
ME-180984781

### CODE REVIEW

File: spec/models/types_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+RSpec.describe Types do
+  describe 'constant FederalPovertyLevels' do
+    def fetch_fpl_data(year)
+      ::Types::FederalPovertyLevels.detect do |fpl_hash|
+        fpl_hash[:medicaid_year] == year
+      end
+    end
+
+    context 'for year 2022' do
+      before do
```
Comment (markgoho) [inline spec/models/types_spec.rb]: what are you testing with these specs? it looks like the spec tests how Ruby works

**Review by markgoho — COMMENTED** [review]
what's the reasoning behind keeping values from 2021 and writing specs for them?

File: spec/models/types_spec.rb (hunk 1 above)
Comment (saikumar9) [reply spec/models/types_spec.rb]: **2021** FPL values will be used for **2022** applications and **2022** FPL values will be used for **2023** applications.

Comment (markgoho) [reply spec/models/types_spec.rb]: understood, that answers why we're keeping these values

now we need to understand what exactly you're testing--it appears that you're testing how Ruby works, the JavaScript equivalent would be:

``` js
const obj = { foo: 'bar' }

expect obj.foo === 'bar'
```

this doesn't do anything meaningful

**Review by markgoho — DISMISSED** [review]

**Review by vkghub — APPROVED** [review]

**Pull request merged**